src/Runner.jl

```diff
@@ -441,7 +441,7 @@ function platform_envs(platform::Platform, src_name::AbstractString; host_platfo
         "target" => target,
         "rust_target" => map_rust_target(platform),
         "rust_host" => map_rust_target(rust_host), # use glibc since musl is broken. :( https://github.com/rust-lang/rust/issues/59302
-        "nproc" => "$(Sys.CPU_THREADS)",
+        "nproc" => "$(get(ENV, "BINARYBUILDER_NPROCS", Sys.CPU_THREADS))",
```

Comment:
Just realised that we have `nproc` and `BINARYBUILDER_NPROCS`, should I remove the trailing `S`?

Reply:
Yes.

Reply:
Done